### PR TITLE
Add actioncable to build step for release

### PR DESCRIPTION
**Pull request by arunagw**
Part of aeceb5d1888496ff2acf429520ea61ea42b79f68

### CODE REVIEW

**Comment by kaspth** [comment]
Let's run CI for this one though? :)

**Comment by arunagw** [comment]
Yes. I have just forced push :-)

**Comment by rafaelfranca** [comment]
We need to update tasks/release.rb too. It would be good if we could avoid this duplication and remove one of the constants

File: Rakefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'net/http'
+require_relative 'tasks/release'
```
Comment (rafaelfranca) [inline Rakefile]: We already have the require in the line 5.

**Comment by arunagw** [comment]
We have already updated that actually.. But we can relative require file and use constant.. But need to double check.. don't want to break any release or test process :-)

**Comment by arunagw** [comment]
Yes. I totally missed that require. So we can directly use `FRAMEWORKS` here

**Pull request merged**